style.css

```diff
@@ -1,3 +1,31 @@
 #root {
   color: red;
+  display: flex;
+  flex-wrap: wrap;
+  width: 75%;
+}
+h1 {
+  font-size: 15px;
+  text-align: center;
+  border: 3px solid grey;
+  border-radius: 10%;
+}
+img {
+  width: 100%;
+}
+.episodeDiv {
+  margin: 20px;
+  width: 25%;
+  padding: 5px;
```

Comment:
Maybe CSS Flexbox could be implemented here instead, as use of margin + padding can alter the design of the page when more features are added.